### PR TITLE
topsql: introduce sql exec count and sql duration

### PR DESCRIPTION
**Pull request by mornyx**
Signed-off-by: mornyx <mornyx.z@gmail.com>

### What is changed and how it works?

- Add two metrics: `sql_exec_count` and `sql_duration`.
- Implement write and query for new metrics.
- Enhance integration tests of TopSQL.

### Related PR

- https://github.com/pingcap/tipb/pull/250

### CODE REVIEW

**Comment by mornyx** [comment]
PTAL, Thanks~
/cc @crazycs520 @zhongzc @breeswish

File: component/topsql/query/default_query.go [hunk 1]
```diff
@@ -123,7 +139,11 @@ func (dq *DefaultQuery) fetchTimeseriesDB(name string, startSecs int, endSecs in
 		log.Warn("failed to fetch timeseries db", zap.String("error", respR.Body.String()))
 	}
 
-	return json.Unmarshal(respR.Body.Bytes(), metricResponse)
+	b := respR.Body.Bytes()
+	if name == store.VirtualMetricNameSQLDuration {
+		fmt.Println(">>>>>", string(b))
+	}
```
Comment (zhongzc) [inline component/topsql/query/default_query.go:145]:
```suggestion
```

**Comment by mornyx** [comment]
Writing and reading for `ExecCount` and `DurationSum` are supported now, we need to support `Duration` on this basis. cc @zhongzc

**Review by zhongzc — APPROVED** [review]

**Pull request merged**